lib/src/message/message_builder.dart

```diff
@@ -247,6 +313,32 @@ class _MessagePartImageAttachmentState extends State<MessagePartImageAttachment>
     );
   }
 
+  void loadThumbnail() async {
+    imageProvider = MemoryImage(kTransparentImage);
+    String videoPath = _getMessageStateData(context).attachmentStateData.path;
+    String thumbnailPath = "";
+    int dotIndex = videoPath.lastIndexOf('.');
+    String videoPathWithoutEnding = videoPath.substring(0, dotIndex);
+    thumbnailPath = "$videoPathWithoutEnding.jpg";
```

Comment:
We should probably do a followup ticket removing all hard coded file types / moving those definitions to a helper / constant file.

Reply:
done in https://github.com/open-xchange/ox-coi/pull/298/commits/da49c93d52a3c3b67204f870a88b4cfe2e5123eb